BeeWare.lektorproject

```diff
@@ -25,7 +25,7 @@ locale = zh_TW
 [alternatives.de]
 name = German
 url_prefix = /de_DE/
```

Comment:
maybe we should just use `/de/` as the url prefix, instead of `/de_DE` ?

Reply:
I would agree but I don't know the policy to this.
But to be fair your proposal has nothing to do with my change and might be better discussed in another issue.

Reply:
Probably, but is a good opportunity to update it :-). @freakboy3742 ?